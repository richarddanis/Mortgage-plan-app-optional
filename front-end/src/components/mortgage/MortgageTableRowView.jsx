import React from "react";
import {TableCell, TableRow} from "@material-ui/core";

import PropTypes from 'prop-types';

const MortgageTableRowView = ({ sentence }) => {
  return (
    <>
      <TableRow>
        <TableCell align="center">
          { sentence }
        </TableCell>
      </TableRow>
    </>
  );
}

MortgageTableRowView.propTypes = {
  sentence: PropTypes.string.isRequired,
};

export default MortgageTableRowView;
