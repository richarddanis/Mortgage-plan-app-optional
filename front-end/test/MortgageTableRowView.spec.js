import React from 'react';
import renderer from 'react-test-renderer';

import MortgageTableRowView from "../src/components/mortgage/MortgageTableRowView";

describe('MortgageTableRowView', () => {
  it('should render', () => {
    const tree = renderer.create(<MortgageTableRowView sentence={'test sentence'}/>).toJSON();

    expect(tree).toMatchSnapshot();
  })
})
