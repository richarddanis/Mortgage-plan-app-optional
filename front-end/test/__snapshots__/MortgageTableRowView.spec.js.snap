// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MortgageTableRowView should render 1`] = `
<tr
  className="MuiTableRow-root"
  role={null}
>
  <td
    aria-sort={null}
    className="MuiTableCell-root MuiTableCell-alignCenter"
  >
    test sentence
  </td>
</tr>
`;
